worker/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "cloudflare-vite-get-started",
+  "private": true,
+  "version": "1.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite dev",
+    "build": "vite build",
+    "preview": "npm run build && vite preview",
+    "deploy": "npm run build && wrangler deploy"
+  },
+  "devDependencies": {
+    "@cloudflare/vite-plugin": "^1.1.1",
+    "@cloudflare/workers-types": "^4.20250507.0",
```

Comment:
Likely can remove this dependency as types are to be generate with `wrangler types` command?